config-wdio/wdio.local.conf.ts

```diff
@@ -54,7 +54,7 @@ export const config: WebdriverIO.Config = {
   // will be called from there.
   //
   specs: [
-    './test/specs/amz.search.specs.ts'
+
```

Comment:
why this spec file is removed here?

Reply:
I am not removed this spec files , i am just editing the specs file name to run the code